drivers/storage/portworx/util/csi_generator.go

```diff
@@ -144,6 +146,14 @@ func (g *CSIGenerator) GetCSIConfiguration() *CSIConfiguration {
 	// Set the CSI driver name
 	cv.DriverName = g.driverName()
 
+	// k8s 1.16 and earlier, CSI registration path should be csi-plugins.
+	// See https://github.com/kubernetes-csi/node-driver-registrar/pull/59/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41
+	if g.kubeVersion.LessThan(k8sVer1_16) {
```

Comment:
Should be less than 1.17

Reply:
https://github.com/portworx/px-installer/commit/686efd72c9feeadb539007a69cd0cf2d755b86a9#diff-deba63de8a1a7f88cc87e2dedff88a3cR148